public/_redirects

```diff
@@ -0,0 +1 @@
+/dashboard/*    /   301!
```

Comment:
I'm not too keen forcing every clone and fork to have this redirect. Perhaps it can be added in the build command as an env variable